src/stories/Forms/InputWrapper/index.tsx

```diff
@@ -54,6 +25,26 @@ const Wrapper = styled.div<WrapperProps>`
     background-color: var(--color-alertRedBg);`}
 `;
 
+const HintWrapper = styled.div<WrapperProps>`
```

Comment:
It looks like this is only using boxyErrorStyle so when using it you can remove hasError:)